### PR TITLE
JAMES-2514 Upgrade Cassandra 3.11.3 -> 3.11.10

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by chibenwa** [comment]
:green_apple: on the CI

**Review by Arsnael — APPROVED** [review]

**Comment by chibenwa** [comment]
Merged

File: docs/modules/servers/pages/distributed/operate/guide.adoc [hunk 1]
```diff
@@ -368,7 +368,7 @@ CREATE ROLE james_one WITH PASSWORD = 'james_one' AND LOGIN = true;
 ==== Create a keyspace
 
 Have a look at
-http://cassandra.apache.org/doc/3.11.3/cql/ddl.html[cassandra documentation] section `CREATE KEYSPACE` for more information
+http://cassandra.apache.org/doc/3.11.10/cql/ddl.html[cassandra documentation] section `CREATE KEYSPACE` for more information
```
Comment (mbaechler) [inline docs/modules/servers/pages/distributed/operate/guide.adoc:371]: this url doesn't work

File: src/site/markdown/server/install/guice-cassandra.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ mvn clean install
 
 ### Requirements
 
- * Cassandra 3.11.3
+ * Cassandra 3.11.10
```
Comment (mbaechler) [inline src/site/markdown/server/install/guice-cassandra.md:28]: If we change the requirement, it means we don't support 3.11.3 any longer and thus we need a breaking change entry, right?

**Comment by chibenwa** [comment]
@mbaechler  see https://github.com/linagora/james-project/pull/4284 for a fix for your comments.

**Pull request closed without merging**